src/pmdas/activemq/ActiveMQ.pm

```diff
@@ -46,7 +46,7 @@ sub queues {
 
     my @queues = @{$query_result};
     my @queue_instances = map {
-	Queue->new($_->{'objectName'}, $self->{_rest_client});
+	PCP::Queue->new($_->{'objectName'}, $self->{_rest_client});
```

Comment:
Thanks!  Is a similar change needed in ActiveMQTest.pl I wonder?  It seems to also use Queue->new()

Reply:
Hi @natoscott ,
I've pushed more changes that should fix all the namespacing issues for the tests.

Reply:
Perfect, thanks Troy.